assets/js/base/context/cart-checkout/validation/index.js

```diff
@@ -178,7 +169,7 @@ export const ValidationContextProvider = ( { children } ) => {
 		hideValidationError,
 		showValidationError,
 		showAllValidationErrors,
-		hasValidationErrors,
+		hasValidationErrors: Object.keys( validationErrors ).length > 0,
```

Comment:
Nice bit of improvement here 👍

Reply:
Kudos to @mikejolley. I think he suggested this improvement some time ago in Slack and this pull was a good opportunity to implement it.